Services.Test/UnitTests/HandCrafted.dbl

```diff
@@ -1616,6 +1616,26 @@ namespace Services.Test.UnitTests
             end
         endmethod
 
+        class DifferentPKWithCount
+            {JsonProperty("@odata.count")}
+            public readwrite property Count, long
+            {JsonProperty("value")}
+            public readwrite property Value, [#]@Services.Models.Differentpk
+        endclass
+
+        {TestMethod}
+        public method DifferentPKJoin, void
```

Comment:
I think we should have a bit more of this. as an example see GetCustomerGeLePhoneRef

Reply:
Added two more tests.
See c032eebba73e134108efa6f12679b870588d8859